source/extensions/filters/http/jwt_authn/authenticator.cc

```diff
@@ -123,6 +126,17 @@ std::string AuthenticatorImpl::name() const {
   return "_UNKNOWN_";
 }
 
+std::string AuthenticatorImpl::convertDoubleOrIntToString(const double& double_value) {
```

Comment:
primitive type, e.g. double, not need to use "const &"  just copy it,  just "double"